.github/workflows/ci.yml

```diff
@@ -29,6 +29,19 @@ jobs:
         run: |
           cargo fmt -- --check
 
+  verusfmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: setup verusfmt
+        run: |
+          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/verus-lang/verusfmt/releases/latest/download/verusfmt-installer.sh | sh
```

Comment:
Pin `verusfmt` version? Otherwise this check could start failing on main branch?

Reply:
Thanks for bringing this up. I've written more about this in https://github.com/verus-lang/verus/pull/1004#issuecomment-1965148204

In summary, it is an explicit decision to _not_ pin it.